packages/desktop/views/dashboard/settings/views/security/Security.svelte

```diff
@@ -1,21 +1,19 @@
 <script lang="ts">
     import { isSoftwareProfile } from '@core/profile'
     import { SecuritySettingsRoute } from '@core/router'
-    import { HR } from 'shared/components'
-    import { AppLock, ChangePassword, ChangePincode, DeleteProfile, ExportStronghold, MaxMediaSize } from './'
     import features from '@features/features'
+    import { HR } from 'shared/components'
+    import { AppLock, ChangePassword, ChangePincode, ExportStronghold } from './'
 
     const settings: {
         component: unknown
         childRoute: SecuritySettingsRoute
         requireSoftware?: boolean
     }[] = [
```

Comment:
Should use interface

Reply:
Existing code that will be refactored when we make these use a generic compoennt.